assignments/this.js

```diff
@@ -1,26 +1,68 @@
 /* The for principles of "this";
-* in your own words. explain the four principle for the "this" keyword below.
-*
-* 1. 
-* 2. 
-* 3. 
-* 4. 
-*
-* write out a code example of each explanation above
-*/
+ * in your own words. explain the four principle for the "this" keyword below.
+ *
+ * 1. Window Binding  - when we use din as a global scobe, you cand bine any variabile, object to the window ex: window.age=3; .. or you can console.log(this);
+ * 2. Implicit Binding - when invok a function by a dot , instead of writing the object we can adreess like  this.function not object.function
+ * 3. New Binding - when we create a object with a contruction function , the object its created and returned without writing the return statement
```

Comment:
When using `new` keyword, `this` will be bound to newly created object at the current time of it's creation.